### PR TITLE
 Move the GDAL Rasterize algorithm to the Vector conversion chapter 

### PR DESCRIPTION
**Pull request by DelazJ**
Updates and adds parameters description (changes are in the last commit)

### CODE REVIEW

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 1]
```diff
@@ -85,37 +85,67 @@ Parameters
...
+  A fixed value to burn into a band for all features.
 
-  Default: *False*
+  Default: *0.0*
 
-``Set output raster size (ignored if above option is checked)`` [enumeration]
-  If the above option is not checked either the output size is in pixels (0)
-  or the output resolution is in map units per pixel (1).
+``Output raster size units`` [enumeration]
+  Units to apply to features.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: Units to use to specify output resolution.

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 2]
```diff
@@ -85,37 +85,67 @@ Parameters
...
 
   Options:
 
-  * 0 --- Output size in pixels
-  * 1 --- Output resolution in map units per pixel
+  * 0 --- Pixels
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: Output resolution is specified using the number of pixels

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 3]
```diff
@@ -85,37 +85,67 @@ Parameters
...
+  * 1 --- Georeferenced units
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: Output resolution is specified using the size of the pixels (in map units)

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 4]
```diff
@@ -85,37 +85,67 @@ Parameters
...
 
-  Default: *1*
+  Default: *0*
 
-``Horizontal`` [number]
+``Width/Horizontal resolution`` [number]
   Sets the horizontal resolution of the output raster.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: Sets the horizontal resolution of the output raster as the number of pixels or the pixels size, depending on the choice of ``Output raster size units``.

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 5]
```diff
@@ -85,37 +85,67 @@ Parameters
...
 
-  Default: *100.0*
+  Default: *0.0*
 
-``Vertical`` [number]
+``Height/Vertical resolution`` [number]
   Sets the vertical resolution of the output raster.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: Sets the vertical resolution of the output raster as the number of pixels or the pixels size, depending on the choice of ``Output raster size units``.

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 6]
```diff
@@ -85,37 +85,67 @@ Parameters
...
 
-  Default: *100.0*
+  Default: *0.0*
+
+``Output extent (xmin, xmax, ymin, ymax)`` [extent]
+  Extent of the output raster layer. If the extent is not specified, the minimum
+  extent that covers selected reference layer(s) will be used.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: covers the selected ...

File: source/docs/user_manual/processing_algs/gdal/vectorconversion.rst [hunk 7]
```diff
@@ -132,14 +162,26 @@ Parameters
...
   * 10 --- CFloat64
 
-  Default: *0*
+  Default: *5*
+
+``Pre-initialize the output image with value`` [number]
+  Optional
+
+  Pre-initializes the output image bands with this value.
+  However, it is not marked as the nodata value in the output file.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/gdal/vectorconversion.rst]: Not marked as the nodata ...

**Comment by DelazJ** [comment]
@havatv Happens that part of this alg description has already been reviewed in #3032.

**Comment by havatv** [comment]
No further comments.

**Pull request merged**